app/assets/javascripts/components/overview/course_cloned_modal.jsx

```diff
@@ -339,7 +339,6 @@ const CourseClonedModal = createReactClass({
           onChange={this.updateCourse}
           value={this.state.course.term}
           value_key="term"
-          required={true}
```

Comment:
We do still want to require it for `wiki_education` mode.

Reply:
Alright i have updated that